src/TerminalVelocityChen2022.jl

```diff
@@ -0,0 +1,111 @@
+module TerminalVelocityChen2022
+
+import SpecialFunctions as SF
+import Thermodynamics as TD
+
+import ..Common as CO
+import ..CommonTypes as CT
+
+import ..Parameters as CMP
+const APS = CMP.AbstractCloudMicrophysicsParameters
+
+export rain_terminal_velocity_individual
+export rain_terminal_velocity_bulk
+
+"""
+    rain_terminal_velocity_individual(ρ, scheme, D_r)
+
+ - `ρ`` - air density
+ - `scheme` - type for parameterization
+ - `D_r` - diameter of the raindrops
+
+ Returns the fall velocity of a raindrop using multiple gamma-function terms
+ to increase accuracy for `scheme == Chen2022Type`
+"""
+function rain_terminal_velocity_individual(
+    ρ::FT,
+    scheme::CT.Chen2022Type,
+    D_r::FT, #in m
+) where {FT <: Real}
+    D_r = D_r * 1000 #D_r is in mm in the paper --> multiply D_r by 1000
+    q = exp(0.115231 * ρ)
+    ai = [0.044612 * q, -0.263166 * q, 4.7178 * q * ρ^(-0.47335)]
+    bi = [2.2955 - 0.038465 * ρ, 2.2955 - 0.038465 * ρ, 1.1451 - 0.038465 * ρ]
+    ci = [0, 0.184325, 0.184325]
+    ϕ = 1
+    v = 0
+    for i in 1:3
+        v = v + (ai[i] * (D_r)^(bi[i]) * exp(-D_r * ci[i]))
+    end
+    return v
+end
+
+"""
+    rain_terminal_velocity(param_set, scheme, q_rai, ρ, N_rai)
+
+ - `param_set` - abstract set with Earth parameters
+ - `scheme` - type for parameterization
+ - `q_rai` - rain water specific humidity
+ - `ρ` - air density
+ - `N_rai` - raindrops number density
+
```

Comment:
What are the units of input/output? Could you also add the link to the Chen2022 paper here?